src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/NoSelectionCriteriaBrowse.feature

```diff
@@ -1,16 +0,0 @@
-﻿Feature: No-selection criteria browse
```

Comment:
where is this covered if you're removing it?

Reply:
All existing non-foundation tests cover this functionality